examples/dora/image-utils/src/lib.rs

```diff
@@ -0,0 +1,89 @@
+use std::collections::BTreeMap;
+
+use arrow::array::PrimitiveArray;
+use arrow::datatypes::UInt8Type;
+use dora_node_api::{ArrowData, IntoArrow, Metadata, Parameter};
+use kornia::image::{Image, ImageSize};
+
+pub fn image_to_arrow(
+    image: Image<u8, 3>,
+    metadata: Metadata,
+) -> eyre::Result<(BTreeMap<String, Parameter>, PrimitiveArray<UInt8Type>)> {
+    let mut meta_parameters = metadata.parameters;
+    meta_parameters.insert("cols".to_string(), Parameter::Integer(image.cols() as i64));
+    meta_parameters.insert("rows".to_string(), Parameter::Integer(image.rows() as i64));
```

Comment:
```suggestion
    meta_parameters.insert("width".to_string(), Parameter::Integer(image.cols() as i64));
    meta_parameters.insert("height".to_string(), Parameter::Integer(image.rows() as i64));
```

Reply:
@haixuanTao I would like to keep kornia nomenclature with rows/cols